src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -75,10 +75,14 @@ public BaseLayout ContentPage
 			{
 				if (value != _ContentPage)
 				{
+					//if (_ContentPage is not null)
+					//	_ContentPage.DirectoryPropertiesViewModel.CheckoutRequested -= GitCheckout_Required;
```

Comment:
Commenting this out seems to have fixed the CI pipeline. Is there any way this would cause a crash?

Reply:
I may try adding a null check when hooking the event

Reply:
It's interesting that this isn't an issue on our local devices.

Reply:
I'm wondering what might be...

Reply:
Thanks for the fix